src/features/RepTokenMigration/helpers.ts

```diff
@@ -0,0 +1,35 @@
+import { DEFAULT_ASSET_DECIMAL, DEFAULT_NETWORK_CHAINID } from '@config';
+import { inputGasPriceToHex, inputValueToHex, toWei } from '@services/EthService';
+import { ERC20, RepV2Token } from '@services/EthService/contracts';
+import { ITokenMigrationFormFull, ITxData, ITxObject } from '@types';
+
+import { repTokenMigrationConfig } from './config';
+
+export const createApproveTx = (payload: ITokenMigrationFormFull): Partial<ITxObject> => {
```

Comment:
Do we not have a generalized formatApproveTx somewhere? If not, can we create one?